src/main/java/GeneralService/OnlineServicesCommunicationService.java

```diff
@@ -0,0 +1,28 @@
+package GeneralService;
+
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OnlineServicesCommunicationService {
+
+    @Autowired
+    private MqttClient mqttClient;
+
+    public void sendRequest(Request request) throws MqttException {
+        if (!mqttClient.isConnected())
+            mqttClient.connect();
+
+        String message = String.valueOf(request.getRequestValue());
+
+        if (request.getRequestType().equals(RequestType.WINDOW_CONTROL_SERVICE_REQUEST) ||
```

Comment:
change this to verify if the incoming request type is a valid RequestType, get the enum values as a list and check if it's present in there